middleware/lib/claim-client.js

```diff
@@ -174,10 +177,17 @@ class ClaimClient {
    * Retrieve a single file for a given claim.
    *
    * @param {string} vin - The VIN of the claim.
-   * @param {Protos.File} fileProto - The file proto object.
+   * @param {string} name - The name of the file to retrieve.
+   * @param {string} hash - The hash of the file to retrieve.
    */
-  async getFile(vin, fileProto) {
-    return s3.getFile(vin, fileProto.name, fileProto.hash);
+  async getFile(vin, name, hash) {
+    const { status } = (await loadType('File')).getEnum('Status');
```

Comment:
This is all incorrect. Lots of things wrong here, I'll talk to you about it in slack.